ci/travis/build-docker-images.py

```diff
@@ -32,11 +32,14 @@
 
 
 def _merge_build():
+    if os.environ.get("BUILDKITE"):
+        return os.environ.get("BUILDKITE_PULL_REQUEST").lower() == "false"
     return os.environ.get("TRAVIS_PULL_REQUEST").lower() == "false"
```

Comment:
Not necessary for this PR, but I honestly think we should switch to passing args to build-docker-images, **not** reading a bunch of env variables